client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1789,7 +1789,18 @@ Undocumented.prototype.updateDns = function( domain, records, fn ) {
 };
 
 Undocumented.prototype.applyDnsTemplate = function( domain, provider, service, variables, callback ) {
-	return this.wpcom.req.post( '/domains/' + domain + '/dns/providers/' + provider + '/services/' + service, { variables }, callback );
+	return this.wpcom.req.post( '/domains/' + domain + '/dns/providers/' + provider + '/services/' + service,
+		{ variables }, callback );
+};
+
+Undocumented.prototype.applyDnsTemplateSyncFlow = function( domain, provider, service, variables, callback ) {
+	return this.wpcom.req.get( '/domain-connect/authorize/v2/domainTemplates/providers/' + provider + '/services/' +
```

Comment:
I was doing it this way to mimic the corresponding path on the WPCOM endpoint which is part of the actual Domain Connect spec, but I can make this one `domain-templates` instead.